system/jlib/jptree.cpp

```diff
@@ -7614,13 +7672,58 @@ static constexpr const char * currentVersion = "1.0";
  *     '__remove__' then that element is removed, otherwise it is merged.  If there is no match it is added.
```

Comment:
Looks like code based on @__remove__ has been removed in this PR, should this comment be removed too?

Reply:
Updated entire comment.